unified-codes/apps/data-service/src/app/resolvers.js

```diff
@@ -34,15 +34,16 @@ export const resolvers = {
       const { dataSources } = context;
       const { code } = _args;
       const query = queries.entity(code);
-      const response = await dataSources.dgraph.postQuery(query);
+      const response = await dataSources.DgraphDataSource.postQuery(query);
```

Comment:
Oops, I think I misnamed these! Should be:

```

Reply:
oh, ok. will update.